### ISSUE TITLE
How to add a new package?

### ISSUE THREAD

**Issue by fwartner** [laravel-notification-channels/new-channels#4]
Have my driver done and tested but can´t figure out, how to make a PR with my provider.. Maybe it´s because it´s too late over here and i´m tired.. ^^

**Comment by themsaid** [comment]
Just clone this repo: https://github.com/laravel-notification-channels/skeleton

And then open a new PR here with your new driver :)

**Comment by gregoriohc** [comment]
It's not better to open an issue with the new channel proposal?
Because PRs here will not be merged and they will always be closed.
By the way, I have a new Twilio channel ready for launch :)

**Comment by fwartner** [comment]
Me too! :D M channel is also for Twilio.. ^^

**Comment by gregoriohc** [comment]
Maybe we need a thread (issue) to make the proposals before we start working on the channels XD

**Comment by fwartner** [comment]
Great idea! Otherwise there would be so many channel twice! :D

**Comment by themsaid** [comment]
hyg https://github.com/laravel-notification-channels/new-channels/issues/6 :)

**Comment by fwartner** [comment]
https://github.com/fwartner/laravel-notification-twilio :)
